WalletWasabi.Fluent/ViewModels/CoinSelection/SelectCoinsDialogViewModel.cs

```diff
@@ -28,43 +26,18 @@ namespace WalletWasabi.Fluent.ViewModels.CoinSelection;
 	NavigationTarget = NavigationTarget.DialogScreen)]
 public partial class SelectCoinsDialogViewModel : DialogViewModelBase<IEnumerable<SmartCoin>>
 {
-	private readonly IObservable<Unit> _balanceChanged;
-	private readonly IEnumerable<SmartCoin> _predefinedCoins;
-	private readonly WalletViewModel _walletViewModel;
+	private readonly CompositeDisposable _disposable = new();
 	private readonly TransactionInfo _transactionInfo;
+	private readonly WalletViewModel _walletViewModel;
 
 	[AutoNotify] private string _searchFilter = "";
-	[AutoNotify] private ReactiveCommand<Unit, Unit> _clearCoinSelectionCommand = ReactiveCommand.Create(() => { });
-	[AutoNotify] private IObservable<bool> _enoughSelected = Observable.Return(false);
-	[AutoNotify] private IObservable<bool> _isSelectionBadlyChosen = Observable.Return(false);
-	[AutoNotify] private LabelBasedCoinSelectionViewModel? _labelBasedSelection;
-	[AutoNotify] private IObservable<Money> _remainingAmount = Observable.Return(Money.Zero);
-	[AutoNotify] private ReactiveCommand<Unit, Unit> _selectAllCoinsCommand = ReactiveCommand.Create(() => { });
-	[AutoNotify] private ReactiveCommand<Unit, Unit> _selectPrivateCoinsCommand = ReactiveCommand.Create(() => { });
-	[AutoNotify] private IObservable<Money> _selectedAmount = Observable.Return(Money.Zero);
-	[AutoNotify] private IObservable<int> _selectedCount = Observable.Return(0);
-	[AutoNotify] private IObservable<Money> _requiredAmount = Observable.Return(Money.Zero);
-	[AutoNotify] private ReactiveCommand<Unit, Unit> _selectPredefinedCoinsCommand = ReactiveCommand.Create(() => { });
-	[AutoNotify] private IObservable<string> _summaryText = Observable.Return("");
 
 	public SelectCoinsDialogViewModel(WalletViewModel walletViewModel, TransactionInfo transactionInfo, IEnumerable<SmartCoin> predefinedCoins)
 	{
 		_walletViewModel = walletViewModel;
 		_transactionInfo = transactionInfo;
-		_balanceChanged = walletViewModel.UiTriggers.BalanceUpdateTrigger;
-		_predefinedCoins = predefinedCoins;
-		
-		SetupCancel(enableCancel: false, enableCancelOnEscape: true, enableCancelOnPressed: false);
-		EnableBack = true;
-		NextCommand = ReactiveCommand.Create(() => new List<ICoin>());
-	}
-
-	private new ReactiveCommand<Unit, List<ICoin>> NextCommand { get; set; }
 
-	[SuppressMessage("Reliability", "CA2000:Dispose objects before losing scope", Justification = "Objects use DisposeWith")]
-	protected override void OnNavigatedTo(bool isInHistory, CompositeDisposable disposables)
-	{
-		var coinChanges = _balanceChanged
+		var coinChanges = walletViewModel.UiTriggers.BalanceUpdateTrigger
```

Comment:
It should be the `TransactionsUpdateTrigger` instead of `BalanceUpdateTrigger` I think